mapreduce/src/main/java/com/aerospike/hadoop/mapreduce/AerospikeConfigUtil.java

```diff
@@ -30,7 +30,7 @@
 
     public static final int DEFAULT_INPUT_PORT = 3000;
     public static final long INVALID_LONG = 762492121482318889L;
-
+    public static final int DEFAULT_INPUT_SCAN_PERCENT = 100;
```

Comment:
default scan percent is 100. If no parameter provided, 100 will be used.

@mzhangsfly